README.md

```diff
@@ -14,12 +14,43 @@ Create a workflow like the one below (also in example directory):
 ```
 
 NOTE:
-The PR commit message wallet address must be seperated by a space " " character from the message.
+The PR commit message wallet address must be seperated by a separator input (default: space) from the message.
 Example:
 initial commit 0x0... (O)
 
 github commit 0x0... message (O)
 
 commit message0x0... (X)
 
+We are able to change the separator with this as an option:
 
+## Custom Separator (Example)
+```yaml
+    with:
+    separator: "\n"
+```
+
+## Invoking the action only when the pull request is merging (Example):
+```yaml
+on:
+  pull_request_target:
+    types:
+      - closed
```

Comment:
Is `closed` the same thing as `merged` (or whatever the equivalent would be?). Asking because it sounds like `closed` is actually just the big button the left 
<img width="206" alt="image" src="https://user-images.githubusercontent.com/15695189/191968902-b7455a66-8858-4f43-b801-fe95b375c0b5.png">